src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/letterdetails/TranslatedOrderRefusalDocumentCollection.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.letterdetails;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class TranslatedOrderRefusalDocumentCollection {
+    private TranslatedOrderRefusalDocument value;
```

Comment:
I believe so because the document template to which this collection is being added and used is expecting the refusal reasons UI text (as opposed to the refusal reason Id which is what's used for serialisation and deserialisation) - this was the way I could think of to include the information in a way the document template is expecting whilst preserving the overall object structure, although I'm open to suggestions here